moderation-commands/infractions.md

```diff
@@ -85,14 +85,14 @@ MoonlightBot requires the following permissions to successfully execute this sub
 
 ## delete
 
-<TODO: describe>
+Delete an infraction
```

Comment:
```suggestion
Deletes an infraction
```

this section could also use a red box stating the irreversibility, refer to reminder page